README.md

```diff
@@ -20,7 +20,7 @@ This capstone consists of the mobile and desktop versions with the following sec
 - JAVASCRIPT
 ## Live Demo
 
-[Live Demo Link]()
+[Live Demo Link](https://whoistolu.github.io/open_source_conference_page/about.html)
```

Comment:
- [ ] **OPTIONAL**
Kindly update your live demo link with the latest updates because the hamburger menu is crashing the website when I click it 😄 🙏 

_Please note that this is a highly suggested request for maintaining a professional repo, and I'm really sorry that I didn't mention it earlier in my review._ 🙏

Reply:
Hello @MahmoudBakr23, 
You earlier said the "Footer homepage doesn't display on the mobile screen".
But the screenshot you just sent shows the desktop screen

Reply:
@Whoistolu I know that! First of all, let me show my appreciation for contacting me ❤️, and second, I know that the design guidelines don't show an exact image for a mobile version with a footer included. However, it's still expected developers to add a footer right! Let's think about it, if we have a footer for a website on our laptops, does it make sense if we opened the very same website on our phones and didn't see the footer? 😄 Please tell me what you think! 🥰

Reply:
![Screenshot from 2021-09-11 09-23-04](https://user-images.githubusercontent.com/58692590/132941631-d98f3f0f-0b1e-4a59-bd64-59935812ae8e.png)
![Screenshot from 2021-09-11 09-23-22](https://user-images.githubusercontent.com/58692590/132941651-14f257d3-1b78-4d79-b592-ce86db013c0d.png)
![Screenshot from 2021-09-11 09-23-43](https://user-images.githubusercontent.com/58692590/132941656-a58f318e-bab7-46f4-a9b2-d3da97c91d2f.png)
![Screenshot from 2021-09-11 09-23-56](https://user-images.githubusercontent.com/58692590/132941662-b8e33b2b-7a32-4dc9-9bba-ff9662af3aa7.png)

![Screenshot from 2021-09-11 09-24-10](https://user-images.githubusercontent.com/58692590/132941670-7400659a-4884-4367-a88c-04f11779b385.png)
![Screenshot from 2021-09-11 09-24-16](https://user-images.githubusercontent.com/58692590/132941681-632e08df-38db-4f7a-9422-75f592e10690.png)

Reply:
Alright, now I understand you want me to add a footer to the homepage mobile version.
Let me implement that.